test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -682,6 +682,116 @@ BOOST_AUTO_TEST_CASE(dev_documenting_no_param_description)
 	expectNatspecError(sourceCode);
 }
 
+BOOST_AUTO_TEST_CASE(user_constructor)
+{
+	char const *sourceCode = R"(
+		contract test {
+			/// @notice this is a really nice constructor
+			function test(uint a, uint second) { }
+		}
+	)";
+
+	char const *natspec = R"ABCDEF({
```

Comment:
What is the `ABCDEF` doing here?

Reply:
It specifies a different delimiter. Otherwise, the string would end early in line 696 after `uint256)"`

Reply:
`ABCDEF` is just a custom delimiter for the raw string literal.

Reply:
Oh right because the quote is right next to it: `)"`